### PR TITLE
Tweak mention bot

### PR DESCRIPTION
**Pull request by wilzbach**
- set more explicit whitelist for assignees
- ping only two reviewers
- wait three days before pinging

Those tweaks should reduce the noise a bit, I am still not sure whether it's wise to hardcode the reviewer list in a public file, but on the other hand this information is public on GitHub anyways (via their profile and commits, PRs, ...)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -64,13 +64,123 @@ github.authenticate({
...
-            "andralex", "DmitryOlshansky", "jmdavis", "rainers", "schveiguy", "CyberShadow"],
-    "dlang/phobos": ["WalterBright", "MartinNowak", "yebblies", "AndrejMitrovic", "klickverbot", "9rnsr",
-            "andralex", "DmitryOlshansky", "jmdavis", "rainers", "schveiguy", "9il", "CyberShadow",
-                "Hackerpilot", "quickfur", "wilzbach"],
+    "dlang/dmd": [
+        "9rnsr",
+        "andralex",
+        "AndrejMitrovic",
+        //"AndrewEdwards",
+        "braddr",
```
Comment (dnadlinger) [inline server.js:72]: We should likely keep Brad blacklisted until we have something that takes age into account.

**Comment by dnadlinger** [comment]
Feel free to merge after removing Brad.

File: server.js (hunk 1 above)
Comment (wilzbach) [reply server.js:72]: He is still on the blacklist (see below)  as we haven't implented a time-based scoring yet.

**Comment by dnadlinger** [comment]
Oh, derp.

**Pull request merged**